fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -858,33 +859,35 @@ public Map<Long, TableIf> getIdToTable() {
 
     public void replayUpdateDbProperties(Map<String, String> properties) {
         dbProperties.updateProperties(properties);
-        binlogConfig = dbProperties.getBinlogConfig();
+        if (PropertyAnalyzer.hasBinlogConfig(properties)) {
+            binlogConfig = dbProperties.getBinlogConfig();
+        }
     }
 
     public boolean updateDbProperties(Map<String, String> properties) throws DdlException {
```

Comment:
when init and update db property replica alloc,  better to  check the system has enough backends ?